cloud/client.go

```diff
@@ -233,13 +239,16 @@ func shouldRetry(status int, body string, err error, warnFunc func(string, ...in
 		}
 		return true
 	}
-	if err != nil {
-		if errors.Cause(err) == ErrNoAuthorizedPublicKeys ||
-			errors.Cause(err) == ErrNoSSHAgent ||
-			strings.Contains(err.Error(), "failed to connect to ssh-agent") {
+	if callErr != nil {
+		causeErr := errors.Cause(callErr)
```

Comment:
Can we use `errors.Is` for these?

Reply:
+1 - apparently this was [added semi-recently in pkg/errors](https://github.com/pkg/errors/releases/tag/v0.9.0) for compatibility with go1.13+ error